### PR TITLE
Add MS teams doc

### PR DESCRIPTION
**Pull request by alyssawada**
add microsoft teams doc to support integration release

### CODE REVIEW

File: docs/sources/chat-options/configure-teams.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+1. Select **Notify by** in the first dropdown and select **Microsoft Teams** in the second dropdown to receive alert notifications in Teams.
+
+### Configure Escalation Chains with Microsoft Teams notifications
+Once your MS Teams integration is configured you can configure Escalation Chains to automatically send alerts from Grafana OnCall to MS Teams channels.
+
+To automatically send alerts to MS Teams channels:
+
+1. Navigate to the **Escalation Chains** tab in Grafana OnCall, select an existing escalation chain or click **+New escalation chain**.
+1. Click the dropdown for **Add escalation step**.
+1. Enable **Post to Microsoft Teams channel** by selecting a connected channel from the dropdown.
```
Comment (matiasb) [inline docs/sources/chat-options/configure-teams.md:64]: Just a note. There is no option to configure Teams at the escalation chains tab. The "Post to Microsoft Teams channel" is available for each integration, when configuring escalations as a first notification step at the integration level.

File: docs/sources/chat-options/configure-teams.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+# Microsoft Teams integration for Grafana OnCall
+
+The Microsoft Teams integration for Grafana OnCall embeds your MS Teams channels directly into your incident response workflow to help your team focus on alert resolution. 
+
+Integrating MS Teams with Grafana OnCall allows users to be notified of alerts directly in MS Teams with automated escalation steps and user notification preferences. Users can also take action on alerts directly from MS Teams, including acknowledge, unacknowledge, resolve, and silence.
+
+## Before you begin
+>NOTE: **This integration is available to Grafana Cloud instances of Grafana OnCall only.**
```
Comment (Matvey-Kuk) [inline docs/sources/chat-options/configure-teams.md:25]: <3

**Review by Matvey-Kuk — APPROVED** [review]

File: docs/sources/chat-options/configure-teams.md (hunk 1 above)
Comment (alyssawada) [reply docs/sources/chat-options/configure-teams.md:64]: Oh! Thanks for the clarification. I'll update that section to reflect that :)

**Review by matiasb — APPROVED** [review]
LGTM

**Pull request merged**